src/main/java/lotto/LottoResult.java

```diff
@@ -0,0 +1,63 @@
+package lotto;
+
+import java.util.*;
+
+public class LottoResult {
+    private final Map<Integer, Integer> resultMap = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Integer, Integer> resultMap = new HashMap<>();
```
Integer, Integer는 어떤 것을 의미하는지 알기 힘들어요~ 어떤 해결 방법이 있을까요?

Reply:
Enum을 활용하면 될 것 같습니다!